ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -519,6 +519,47 @@ objects:
           libvirt:
             URI: ${remote_libvirt_uri}
             network:
+              dnsmasqOptions:
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0/${BASTION_SVC_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-0/${BASTION_SVC_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-1/${BASTION_SVC_0_1_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-1/${BASTION_SVC_0_1_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-2/${BASTION_SVC_0_2_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-2/${BASTION_SVC_0_2_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-3/${BASTION_SVC_0_3_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-3/${BASTION_SVC_0_3_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-4/${BASTION_SVC_0_4_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-0-4/${BASTION_SVC_0_4_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-0/${BASTION_SVC_1_0_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-0/${BASTION_SVC_1_0_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-1/${BASTION_SVC_1_1_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-1/${BASTION_SVC_1_1_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2/${BASTION_SVC_1_2_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-2/${BASTION_SVC_1_2_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3/${BASTION_SVC_1_3_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-3/${BASTION_SVC_1_3_IP}"
+              - name: "address"
+                value: "api.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4/${BASTION_SVC_1_4_IP}"
+              - name: "address"
+                value: "/.apps.${LEASED_RESOURCE}-${JOB_NAME_HASH}.${CLUSTER_TYPE}-1-4/${BASTION_SVC_1_4_IP}"
```

Comment:
I think this should just be 1 consistent address which should work for all clusters:
- name: "address"
  value: "/.apps.${CLUSTER_NAME}.${LEASED_RESOURCE}/192.168.${cluster_subnet}.1"

You don't want to set the api.x value since that is set in the network already to round-robin between the 3 master nodes setup in the cluster directly.

Reply:
agreed. from a technical standpoint point too, the dnsmasq options are passed in as a key value mapping, so specifying multiple will not work.